Doc/library/collections.rst

```diff
@@ -1175,6 +1176,11 @@ attribute.
     subclass) or an arbitrary sequence which can be converted into a string using
     the built-in :func:`str` function.
 
+    .. attribute:: data
```

Comment:
Shouldn't be added the paragraph "In addition to supporting ..." as for UserDict and UserList?